test/Serilog.Tests/Context/LogContextTests.cs

```diff
@@ -143,7 +144,7 @@ public void MultipleNestedPropertiesOverrideLessNestedOnes()
         // since user property types may not be serializable.
         // Fails if the Serilog assemblies cannot be loaded in the
         // remote domain. See also LogContext.Suspend()
-        [Test]
+        [Fact(Skip = "Fails on DNX451")]
```

Comment:
Yes, but there could be several reasons for why this fails now. It could be the migration to xUnit (I know they run tests in parallel), or the fact that it runs through DNX.